src/Microsoft.Health.Fhir.Api/Controllers/AadSmartOnFhirProxyController.cs

```diff
@@ -389,23 +389,27 @@ public AadSmartOnFhirProxyController(IOptions<SecurityConfiguration> securityCon
             tokenResponse["client_id"] = clientId;
 
             // Replace fully qualifies scopes with short scopes and replace $
-            string[] scopes = tokenResponse["scope"].ToString().Split(' ');
-            var scopesBuilder = new StringBuilder();
+            string[] scopes = tokenResponse["scope"]?.ToString().Split(' ');
 
-            foreach (var s in scopes)
+            if (scopes != null)
```

Comment:
Should this throw a bad request exception?

Reply:
Apparently, it is OK to have a token response with no scopes (if none were requested), in which case, we should just not do anything with them. It is not an error on the AAD side and the token is valid, so I think this is OK? Any concens?

Reply:
Ok, makes sense. I just wanted to make sure that it wasn't an issue because we throw bad requests elsewhere in this code.